Therapeut Vechter/Assets/Scripts/UI/PlayTutorial.cs

```diff
@@ -0,0 +1,62 @@
+using FMOD.Studio;
+using FMODUnity;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PlayTutorial : MonoBehaviour
+{
+    private EventInstance dialogueAudioEventInstance;
```

Comment:
not the biggest issue, but sorting all your variables would be nice, many ways of doing it, main recommendation is a section for a section of public variables, then serializefields then privates